nntrainer/tensor/tensor_v2.cpp

```diff
@@ -151,6 +151,31 @@ TensorV2 &TensorV2::apply(std::function<TensorV2 &(TensorV2, TensorV2 &)> f,
   return f(*this, output);
 }
 
+int TensorV2::multiply_i(float const &value) { return ML_ERROR_NONE; }
+
+TensorV2 TensorV2::multiply(float const &value) const {
+  TensorV2 t;
+  return multiply(value, t);
+}
+
+TensorV2 &TensorV2::multiply(float const &value, TensorV2 &out) const {
+  return out;
+}
+
+int TensorV2::multiply_i(TensorV2 const &m, const float beta) {
+  return ML_ERROR_NONE;
+}
+
+TensorV2 TensorV2::multiply(TensorV2 const &m, const float beta) const {
+  TensorV2 t("", this->getFormat());
```

Comment:
Looking at Tensor's Basic Constructor, the default value of name_ is "".
Isn't it possible to omit it?

https://github.com/nnstreamer/nntrainer/blob/477a8aa065b0bd7bb885c87ac752ab508497a870/nntrainer/tensor/tensor_v2.h#L38

Reply:
discussed it in person. resolved.